packages/zoe/src/types.js

```diff
@@ -349,17 +282,82 @@
  * @property {Issuer} issuer
  * @property {AmountMath} amountMath
  *
- * @typedef {Object} InstallationRecord
- * @property {InstallationHandle} handle - opaque identifier for the installation
- * @property {SourceBundle} bundle - contract code
- *
- * @typedef {Object} OfferStatus
- * @property {OfferHandle[]} active
- * @property {OfferHandle[]} inactive
- *
- * @typedef {(offerHandle: OfferHandle) => void} OfferHook
- *
- * @typedef {Keyword[]} SparseKeywords
  * @typedef {{[Keyword:string]:Amount}} Allocation
  * @typedef {{[Keyword:string]:AmountMath}} AmountMathKeywordRecord
  */
+
+/**
+ * @typedef {Object} ZCFSeat
+ * @property {() => void} exit
+ * @property {(msg: string=) => never} kickOut
+ * @property {() => Notifier<Allocation>} getNotifier
+ * @property {() => boolean} hasExited
+ * @property {() => ProposalRecord} getProposal
+ * @property {(keyword: Keyword, brand: Brand) => Amount} getAmountAllocated
+ * The brand is used for filling in an empty amount if the `keyword`
+ * is not present in the allocation
+ * @property {() => Allocation} getCurrentAllocation
+ * @property {(newAllocation: Allocation) => Boolean} isOfferSafe
+ * @property {(newAllocation: Allocation) => StagedSeat} stage
+ */
+
+/**
+ * @typedef {Object} StagedSeat
```

Comment:
Now that it has-a seat rather than is-a seat, we should find a name other than `<adjective>Seat`. The first that occurs to be is to flip it: `SeatStaging`. It is a staging object about a seat.

Reply:
I like that!